internal/simplecue/utils.go

```diff
@@ -177,8 +177,30 @@ func cueConcreteToScalar(v cue.Value) (interface{}, error) {
 
 		return values, nil
 	case cue.StructKind:
-		// TODO: this would be used for struct-level default values
-		return nil, nil //nolint: nilnil
+		newMap := make(map[string]interface{})
+		iter, _ := v.Fields(cue.Optional(true), cue.Definitions(true))
```

Comment:
What's the second return value of `v.Fields()`? 
ie: are we silencing an error here? :scream:

Reply:
I omitted it because we are inside struct clause.

Reply:
And that means we don't have to deal we errors? :thinking: (I'm not super super familiar with how cue works, so it's a real question :D)

Reply:
It returns an error if it isn't a struct:
```
// Fields creates an iterator over v's fields if v is a struct or an error
// otherwise.
```